drivers/wdm/audio/legacy/wdmaud/mmixer.c

```diff
@@ -817,7 +824,6 @@ CreatePinCallback(
         ZwClose(*PinHandle);
         return MM_STATUS_UNSUCCESSFUL;
     }
-
```

Comment:
readd this newline. no need for cosmetic changes here.

Reply:
Fixed (I think).

Reply:
no it's not fixed. now that line contains unintended spaces that were not there before.

Reply:
How do I fix it?

Reply:
Put your cursor behind the closing curly brace and press enter, then save the file.
But: Check if your editor has some that may automatically removes trailing newlines.

Reply:
I am using Open Watcom Text Editor.

Reply:
What lines do I delete?